src/main/kotlin/org/rust/lang/core/macros/decl/MacroGraph.kt

```diff
@@ -42,7 +42,7 @@ sealed class MGNodeData : PresentableNodeData {
 typealias MacroGraph = PresentableGraph<MGNodeData, Unit>
 typealias MacroGraphNode = Node<MGNodeData, Unit>
 
-class MacroGraphBuilder(private val macro: RsMacro) {
+class MacroGraphBuilder(private val macro: RsMacroDefinitionBase) {
```

Comment:
As I can see, changes in `MacroGraphBuilder` and `Matcher` are not related to name resolution, It's about completion when macro cannot be expanded properly that @ortem introduced in #4001
It seems that it really makes completion work in calls of macro 2.0 but there are no tests for it.
If I'm right, maybe extract these changes into separate PR and add the corresponding tests (see `org.rust.lang.core.completion.RsPartialMacroArgumentCompletionTest`)?

Reply:
Thanks, I removed code related to `MacroGraphBuilder`, and will open separate PR for it after this PR will be merged